src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REGEX_TO_SPLIT = ",";
+    private static final String CONTENTS = "fruit, quantity";

```

Comment:
The constant `CONTENTS` represents the header of the report. A more descriptive name such as `HEADER` may improve the readability of the code.